### PR TITLE
6715 domain summary display integrated

### PR DESCRIPTION
**Pull request by wschaeferB**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/discovery/ui/DomainWrapper.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.discovery.ui;
+
+import java.awt.Image;
+import org.sleuthkit.autopsy.discovery.search.ResultDomain;
+
+/**
```
Comment (dannysmyda) [inline Core/src/org/sleuthkit/autopsy/discovery/ui/DomainWrapper.java:24]: Missing header

File: Core/src/org/sleuthkit/autopsy/discovery/ui/ResultsPanel.java [hunk 2]
```diff
@@ -116,6 +120,7 @@
                 }
             }
         });
+        //JIRA-TODO
```
Comment (dannysmyda) [inline Core/src/org/sleuthkit/autopsy/discovery/ui/ResultsPanel.java:123]: Was there something you wanted to do here?

File: Core/src/org/sleuthkit/autopsy/discovery/ui/DomainSummaryViewer.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.discovery.ui;
+
+import javax.swing.DefaultListModel;
+
+/**
```
Comment (dannysmyda) [inline Core/src/org/sleuthkit/autopsy/discovery/ui/DomainSummaryViewer.java:23]: Missing header

**Review by dannysmyda — CHANGES_REQUESTED** [review]

File: Core/src/org/sleuthkit/autopsy/discovery/ui/ResultsPanel.java [hunk 4]
```diff
@@ -763,4 +791,44 @@ protected void done() {
...
+            try {
+                get();
+            } catch (InterruptedException | ExecutionException ex) {
+                domainWrapper.setThumnail(null);
+                logger.log(Level.WARNING, "Document Worker Exception", ex);
+            } catch (CancellationException ignored) {
+                domainWrapper.setThumnail(null);
+                //we want to do nothing in response to this since we allow it to be cancelled
+            }
+            documentPreviewViewer.repaint();
```
Comment (dannysmyda) [inline Core/src/org/sleuthkit/autopsy/discovery/ui/ResultsPanel.java:830]: Cut and paste error

File: Core/src/org/sleuthkit/autopsy/discovery/ui/DomainWrapper.java [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        this.domain = domain;
+    }
+
+    /**
+     * Set the thumbnail which exists.
+     *
+     * @param thumbnail The Image object which will be used to represent this
+     *                  domain object.
+     */
+    void setThumnail(Image thumbnail) {
```
Comment (dannysmyda) [inline Core/src/org/sleuthkit/autopsy/discovery/ui/DomainWrapper.java:49]: Typo

File: Core/src/org/sleuthkit/autopsy/discovery/ui/ResultsPanel.java (hunk 2 above)
Comment (wschaeferB) [reply Core/src/org/sleuthkit/autopsy/discovery/ui/ResultsPanel.java:123]: Yes there was I went looking for the JIRA and got distracted, will fix that.

File: Core/src/org/sleuthkit/autopsy/discovery/ui/DomainWrapper.java (hunk 1 above)
Comment (wschaeferB) [reply Core/src/org/sleuthkit/autopsy/discovery/ui/DomainWrapper.java:24]: thank you

File: Core/src/org/sleuthkit/autopsy/discovery/ui/ResultsPanel.java (hunk 4 above)
Comment (wschaeferB) [reply Core/src/org/sleuthkit/autopsy/discovery/ui/ResultsPanel.java:830]: yes it is

File: Core/src/org/sleuthkit/autopsy/discovery/ui/DomainSummaryViewer.java (hunk 3 above)
Comment (wschaeferB) [reply Core/src/org/sleuthkit/autopsy/discovery/ui/DomainSummaryViewer.java:23]: thank you

File: Core/src/org/sleuthkit/autopsy/discovery/ui/DomainWrapper.java (hunk 5 above)
Comment (wschaeferB) [reply Core/src/org/sleuthkit/autopsy/discovery/ui/DomainWrapper.java]: Well i'll b

**Review by dannysmyda — APPROVED** [review]

**Pull request merged**